CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ All notable changes to this project will be documented in this file.
 
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## 1.3.2 (3. July 2019)
++ Fixed bug when header contains (partial) formatting and line before header contains formatting
```

Comment:
Please refer to the pull request `https://github.com/nadar/quill-delta-parser/pull/15` like we did in previous changelog entries. Just take a look at the other entries, it will be self explained. Thanks

Reply:
Cool - first time i'm contributing, so thanks for your patience :)